test/assertion.js

```diff
@@ -347,6 +347,25 @@ describe('Assertion', function(){
     });
   });
 
+  describe('.error(status, fn)', function(){
```

Comment:
the documentation clearly states:

> optional error `msg` or `status`.

but yeah, if you think we need both, I can add support